_includes/body-footer.html

```diff
@@ -1,6 +1,7 @@
 <div class='wrapper--footer js-wrapper'>
-<div class='row row--sitemap' id='js-footer-nav'>
-<div class='row__inner row__inner--footer row--sitemap__inner'>
+<nav aria-label='Sitemap' class='row row--sitemap' id='js-footer-nav' itemscope='itemscope' itemtype='http://schema.org/SiteNavigationElement' role='navigation'>
```

Comment:
Should `js-footer-nav` be a class? I only ask because the `js-*` is a class on other elements.